model/user.go

```diff
@@ -884,24 +885,39 @@ func UsersWithGroupsAndCountFromJson(data io.Reader) *UsersWithGroupsAndCount {
 	return uwg
 }
 
-var passwordRandomSource = rand.NewSource(time.Now().Unix())
+type lockedSource struct {
+	mu  sync.Mutex
+	src rand.Source
+}
+
+func (r *lockedSource) Intn(n int) int {
+	r.mu.Lock()
+	rn := rand.New(r.src)
```

Comment:
I would expect to create the generator (`rand.*Rand`) only once and use it as needed.
Here instead we create a new one every time we need to generate a random integer. Is there a reason for that?

Reply:
No specific reason, it was used like this way so, I adopted that approach. It can be removed.

Reply:
Before the generator was created only once on top of `GeneratePassword` and then used.
Here we are essentially creating a brand new one for each `Intn` call.

Reply:
Yeah 🤦